mods/farming/api.lua

```diff
@@ -130,7 +130,7 @@ local function tick_again(pos)
 end
 
 -- Seed placement
-farming.place_seed = function(itemstack, placer, pointed_thing, plantname)
+farming.place_seed = function(itemstack, placer, pointed_thing, plantname, soil_group)
```

Comment:
Why make this a param rather than doing `local soil_group = farming.registered_plants[plantname].soil_group`?